packages/geo/src/lib/filter/ogc-filterable-item/ogc-filterable-item.component.ts

```diff
@@ -283,4 +294,11 @@ export class OgcFilterableItemComponent implements OnInit {
   public setVisible() {
     this.layer.visible = true;
   }
+
+  changeOgcFilterType(isAdvancedOgcFilters) {
+    this.datasource.options.ogcFilters.advancedOgcFilters = isAdvancedOgcFilters.checked;
```

Comment:
We don't want to update options directly. Create a setter or a method to achieve that.

Reply:
I've created a method.